resources/fra.ini

```diff
@@ -0,0 +1,23 @@
+; Updated time : 15th 11 2017
+; general - Français 
+divider = "========================================"
+spacer = " -=+=- {%0} -=+=- "
+error = "Quelque chose c'est mal passer!"
+noperm = "Tu n'a pas la permission d'utilisé cette commande."
+runingame = "S'il vous plait utilisé cette commande en jeu seulement!"
+commands = "commandes"
+; user interfaces
+ui.confirmation = "Confirmation"
+ui.confirmation.yes = "Oui"
+ui.confirmation.no = "Non"
+; ui ban
+ui.brush.title = "Brush menu"
+ui.brush.options.type.title = "Type"
+ui.brush.options.type.sphere = "Sphère"
+ui.brush.options.type.cylinder = "Cylindre"
+ui.brush.options.type.square = "Carré"
+ui.brush.options.diameter = "Diamètre/Largeur"
+ui.brush.options.height = "Hauteur"
+ui.brush.options.blocks = "Blocs"
+ui.brush.options.blocks.placeholder = "Blocs séparer par des points-virgules"
+ui.brush.options.label.infoapply = "Clique le button "Submit" pour appliquer"
```

Comment:
`"Submit"` will not work in this case because the full line is enclosed in `"` - please either change the outer ones to `'` or add `\` infront of the center ones